src/FileEntityInterface.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\file_entity\FileEntityInterface.
+ */
+
+namespace Drupal\file_entity;
+
+/**
+ * File entity interface.
+ */
+interface FileEntityInterface {
+  /**
+   * Get the metadata property value for the given property.
+   *
+   * @param string $property
+   *   A metadata property for the file.
+   *
+   * @return int|null
+   *   A metadata property value.
+   */
+  public function getMetadata($property);
+
+  /**
+   * Returns whether or not metadata property exists.
```

Comment:
Comments for this kind of functions usually start with "Determines...." or "Checks....".